### PR TITLE
Add validateEmail 

### PR DESCRIPTION
**Pull request by Sajma27**
# Summary
A description of the changes proposed in the pull request.

Fixes #number

## Checklist
- [ ] Change is small and easy to review (please split big changes into multiple PRs)
- [ ] Change is consistent with architecture
- [ ] Change is consistent with test architecture
- [ ] Change is consistent with naming conventions
- [ ] New code is covered with tests
- [ ] Tests related to old code are updated
- [ ] Documentation is up to date with changes

### CODE REVIEW

File: universal-login-commons/src/core/utils/validateEmail.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const validateEmail = (email: string) => {
```
Comment (xitronix) [inline universal-login-commons/src/core/utils/validateEmail.ts:1]: add test

Comment (Sajma27) [reply universal-login-commons/src/core/utils/validateEmail.ts:1]: Done

Comment (xitronix) [reply universal-login-commons/src/core/utils/validateEmail.ts:1]: if it validateEmail should throw error, else it should be 'isProperEmail'

File: universal-login-commons/test/core/utils/validateEmail.test.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import {validateEmail} from '../../../src/core/utils/validateEmail';
+import {expect} from 'chai';
+
+describe('UNIT: ValidateEmail', () => {
+  it('Should validate valid email', () => {
+    expect(validateEmail('valid@email.com')).be.eq(true);
+  });
```
Comment (xitronix) [inline universal-login-commons/test/core/utils/validateEmail.test.ts:7]: Wrige more tests. You can find more edgecases :
```
it('correct email',  () => {
 ...'
}

describe('invalid emal', () => {
  it('empty string', () => {
    validate('')..
}
'e@email'
'@email'
'somestring'
'.account@email.com'
'_account@email.com'
....
}
```

Comment (xitronix) [reply universal-login-commons/test/core/utils/validateEmail.test.ts:7]: For creating test u can use at least these rules: https://www.itsupportguides.com/knowledge-base/exchange-2010/email-address-rules/

Comment (Sajma27) [reply universal-login-commons/test/core/utils/validateEmail.test.ts:7]: Done

File: universal-login-commons/src/core/utils/validateEmail.ts (hunk 1 above)
Comment (Sajma27) [reply universal-login-commons/src/core/utils/validateEmail.ts]: Done

File: universal-login-commons/test/core/utils/validateEmail.test.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import {isProperEmail} from '../../../src/core/utils/isProperEmail';
+import {expect} from 'chai';
+
+describe('UNIT: ValidateEmail', () => {
```
Comment (xitronix) [inline universal-login-commons/test/core/utils/validateEmail.test.ts:4]: UNIT: isProperMail
rename file too

Comment (Sajma27) [reply universal-login-commons/test/core/utils/validateEmail.test.ts:4]: Done

File: universal-login-commons/test/core/utils/validateEmail.test.ts [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  it('Should validate valid email', () => {
+    expect(isProperEmail('valid@email.com')).be.eq(true);
+  });
+
+  describe('Should not validate invalid email', () => {
```
Comment (xitronix) [inline universal-login-commons/test/core/utils/validateEmail.test.ts]: Does this function validate smth or return false? Name this test: 'invalid email'

File: universal-login-commons/test/core/utils/validateEmail.test.ts [hunk 5]
```diff
@@ -0,0 +1,36 @@
+import {isProperEmail} from '../../../src/core/utils/isProperEmail';
+import {expect} from 'chai';
+
+describe('UNIT: ValidateEmail', () => {
+  it('Should validate valid email', () => {
```
Comment (xitronix) [inline universal-login-commons/test/core/utils/validateEmail.test.ts]:
```suggestion
  it('correct email', () => {
```

File: universal-login-commons/test/core/utils/isProperMail.test.ts [hunk 6]
```diff
@@ -0,0 +1,36 @@
+import {isProperEmail} from '../../../src/core/utils/isProperEmail';
+import {expect} from 'chai';
+
+describe('UNIT: isProperMail', () => {
+  it('Should validate valid email', () => {
```
Comment (xitronix) [inline universal-login-commons/test/core/utils/isProperMail.test.ts:5]:
```suggestion
  it('correct email', () => {
```

File: universal-login-commons/test/core/utils/isProperMail.test.ts [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+    expect(isProperEmail('valid@email.com')).be.eq(true);
+  });
+
+  describe('Should not validate invalid email', () => {
```
Comment (xitronix) [inline universal-login-commons/test/core/utils/isProperMail.test.ts:9]:
```suggestion
  describe('invalid email', () => {
```

**Pull request merged**